File: cpp/bc1115.cpp
#include <iostream>

using namespace std;

int main()
{
	int x, y;
	string msg = "";
	
	while(true)
	{
		cin >> x >> y;
		
		if(x == 0 || y == 0) break;
		else
		{
			if(x > 0 && y > 0) msg = "primeiro";
			else if(x < 0 && y > 0) msg = "segundo";
			else if(x < 0 && y < 0) msg = "terceiro";
			else msg = "quarto";
		}
		cout << msg << endl;
	}
}

File: python/bc1115.py
while True:
    msg = ""
    cVal = input().split()

    x = int(cVal[0])
    y = int(cVal[1])

    if x == 0 or y == 0: break
    else:
        fVer = x >= 1
        sVer = y >= 1

        if fVer == True and sVer == True: msg = "primeiro"
        elif fVer == False and sVer == True: msg = "segundo"
        elif fVer == False and sVer == False: msg = "terceiro"
        else: msg = "quarto"

        print(msg)